components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.common/src/main/java/org/wso2/carbon/identity/api/server/configs/common/ConfigsServiceHolder.java

```diff
@@ -52,4 +54,24 @@ public static void setApplicationManagementService(ApplicationManagementService
 
         ConfigsServiceHolder.applicationManagementService = applicationManagementService;
     }
+
+    /**
+     * Get IdentityProviderManager osgi service.
+     *
+     * @return IdentityProviderManager
+     */
+    public static IdentityProviderManager getIdentityProviderManager() {
```

Comment:
Remove static... class is defined as a Singleton

Reply:
addressed